src/shared/components/NotFound.tsx

```diff
@@ -169,6 +174,13 @@ const NotFound: FC = () => {
     }
   }, [handleDeepLink])
 
+  useEffect(() => {
+    dispatch(setCurrentPage('not found'))
+    return () => {
+      dispatch(setCurrentPage('not set'))
```

Comment:
just to clarify, is this return statement here to set the `currentPage` state to `not set` right after the component unmounts?

Reply:
Yup! You nailed it @blegesse-git 

This is also the way we currently handle it on the dashboards page.

Reply:
Basically - this ensures we get a redux state update whenever the user clicks away from the "Not Found" page - whether it's through navigating to a new url, clicking a link on the navbar, etc.

Reply:
thanks for the explanation! makes perfect sense